src/js/media-playback-rate-button.js

```diff
@@ -70,12 +79,42 @@ class MediaPlaybackRateButton extends MediaChromeButton {
     super.attributeChangedCallback(attrName, oldValue, newValue);
   }
 
+  /**
+   * @type {Array<number>} The possible playback rates
+   */
+  get rates() {
```

Comment:
issue(blocker): Needs to be refactored to use `AttributeTokenList` (type `DOMTokenList`)